src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-sidebar/index.ts

```diff
@@ -570,6 +570,8 @@ export default Shopware.Component.wrapComponentConfig({
             if (!dragSectionHasBlock && !dropSectionHasBlock) {
                 this.page.sections![dragSectionIndex].blocks!.add(dragData.block);
             }
+
+            this.$emit('page-save');
```

Comment:
Does this mean that everytime you drag something, but abort the process the page will be saved? That doesn't sound that right to me 🤔 Do I misunderstand something?